TLM/TLM/TrafficManagerMod.cs

```diff
@@ -94,6 +94,12 @@ public class TrafficManagerMod : IUserMod {
             InGameHotReload = InGame();
 
             HarmonyHelper.EnsureHarmonyInstalled();
+
+#if DEBUG
+            const bool installHarmonyASAP = false; // set true for fast testing
```

Comment:
I don't like the solution but I guess its better than nothing for the time beeing.
So i m fine with that.